dags/partybal.py

```diff
@@ -0,0 +1,28 @@
+from airflow import DAG
+from datetime import timedelta, datetime
+from operators.gcp_container_operator import GKEPodOperator
+
+default_args = {
+    "owner": "ascholtz@mozilla.com",
+    "email": ["ascholtz@mozilla.com", "msamuel@mozilla.com",],
+    "depends_on_past": False,
+    "start_date": datetime(2021, 6, 21),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+with DAG("partybal", default_args=default_args, schedule_interval="0 */3 * * *") as dag:
```

Comment:
Does scheduling every 3 hours seem right?

Reply:
sgtm 🙌 

Right now the docker image defaults to rebuilding everything that changed within the last 25 hours. Usually rebuilding an experiment is not expensive -- it's just time, and that's not critical here, but we could shorten the lookback interval if we wanted.

It does mean you might have to run partybal manually if it missed an update.

Reply:
I guess the schedule can be easily adjusted if we see a need